hier_motif_feature_generator.py

```diff
@@ -89,13 +89,13 @@ def get_possible_motifs_to_targets(self, mask_shape):
                 theta_mask[i, center_nucleotide_idx] = False
         return theta_mask
 
-    def combine_thetas_and_get_conf_int(self, theta, sample_obs_info=None, col_idx=0, zstat=ZSCORE_95, add_targets=True):
+    def combine_thetas_and_get_conf_int(self, theta, possible_theta_mask, sample_obs_info=None, col_idx=0, zstat=ZSCORE_95, add_targets=True):
         """
         Combine hierarchical and offset theta values
         """
         full_feat_generator = MotifFeatureGenerator(
             motif_len=self.motif_len,
-            distance_to_start_of_motif=-self.left_motif_flank_len,
+            distance_to_start_of_motif= -self.left_motif_flank_len[0][0],
```

Comment:
i updated this function since it didn't seem to be working. can you check that im using the feature generators correctly and such? @dawahs